dashboard/components/RelationDependencyGraph.tsx

```diff
@@ -114,22 +118,59 @@ export default function RelationDependencyGraph({
 
     const isSelected = (id: string) => id === selectedId
 
-    const applyEdge = (sel: EdgeSelection) =>
+    const applyEdge = (sel: EdgeSelection) => {
+      const color = (d: Edge) => {
+        if (backPressures) {
+          let value = backPressures.get(`${d.target}_${d.source}`)
+          if (value) {
+            return backPressureColor(value)
+          }
+        }
+
+        return theme.colors.gray["300"]
+      }
+
+      const width = (d: Edge) => {
+        if (backPressures) {
+          let value = backPressures.get(`${d.target}_${d.source}`)
+          if (value) {
+            return backPressureWidth(value)
+          }
+        }
+
+        return 2
+      }
+
       sel
         .attr("d", ({ points }) => line(points))
         .attr("fill", "none")
-        .attr("stroke-width", 1)
-        .attr("stroke-width", (d) =>
-          isSelected(d.source) || isSelected(d.target) ? 4 : 2
-        )
+        .attr("stroke-width", width)
+        .attr("stroke", color)
         .attr("opacity", (d) =>
           isSelected(d.source) || isSelected(d.target) ? 1 : 0.5
         )
-        .attr("stroke", (d) =>
-          isSelected(d.source) || isSelected(d.target)
-            ? theme.colors.blue["500"]
-            : theme.colors.gray["300"]
-        )
```

Comment:
nit: the display of `isSelected` state becomes less clear.

new:
<img width="866" alt="image" src="https://github.com/user-attachments/assets/676750a1-49f6-422d-84e4-faef27cbacf3">

old:
<img width="780" alt="image" src="https://github.com/user-attachments/assets/1ff32519-0381-4a0a-8ff8-3cda38b77b15">

Reply:
It's a reasonable trade-off. Edges still have opacity applied. So for edges with non-zero backpressure i.e. color green/red, they will show as more prominent. But for edges without BP, the problem is present but reasonable.